Apps/Playground/Scripts/experience.ts

```diff
@@ -335,27 +354,27 @@ CreateBoxAsync(scene).then(function () {
                     // Listen for changes in feature points both being added and updated, and only update
                     // our display every 60 changes to the feature point cloud to avoid slowdowns.
                     let featurePointChangeCounter = 0;
-                    xrFeaturePointsModule.onFeaturePointsAddedObservable.add((addedPointIds) => {
+                    xrFeaturePointsModule.onFeaturePointsAddedObservable.add((addedPointIds: number[]): void => {
                         if (++featurePointChangeCounter % 60 == 0) {
                             pcs.setParticles();
                         }
                     });
 
-                    xrFeaturePointsModule.onFeaturePointsUpdatedObservable.add((updatedPointIds) => {
+                    xrFeaturePointsModule.onFeaturePointsUpdatedObservable.add((addedPointIds: number[]) => {
                         if (++featurePointChangeCounter % 60 == 0) {
                             pcs.setParticles();
                         }
                     });
                 }
 
-                let sessionMode = vr ? "immersive-vr" : "immersive-ar"
+                let sessionMode: XRSessionMode = vr ? "immersive-vr" : "immersive-ar";
                 if (hololens) {
-                    // Because HoloLens 2 is a head mounted display, its Babylon.js immersive experience more closely aligns to vr
+                    // Because HoloLens 2 is a head mounted display, its js immersive experience more closely aligns to vr
```

Comment:
Is this intentional? Maybe a find and replace?

Reply:
indeed, find and replace. fixed.